lib/python/qmk/cli/list/keymaps.py

```diff
@@ -1,25 +1,29 @@
-"""List the keymaps for a specific keyboard
+""" List the keymaps for a specific keyboard
 """
-from milc import cli
 
 import qmk.keymap
-from qmk.decorators import automagic_keyboard
+from milc import cli
 from qmk.errors import NoSuchKeyboardError
+from qmk.decorators import automagic_keyboard
 
 
-@cli.argument("-kb", "--keyboard", help="Specify keyboard name. Example: 1upkeyboards/1up60hse")
-@cli.subcommand("List the keymaps for a specific keyboard")
+@cli.argument('-kb', '--keyboard', help='Specify the keyboard. Example: 1upkeyboards//1up60hse')
+@cli.subcommand('List the keymaps for a specific keyboard')
 @automagic_keyboard
 def list_keymaps(cli):
     """List the keymaps for a specific keyboard
     """
+
+    # Prompt the user for a keyboard
+    while not cli.args.keyboard:
```

Comment:
Most of the time, you want to use `qmk.config.<submodule>.<key>` as some blackmagic is going on inside `milc`.
Basically it merges together the config, the environment and the user input.
`cli.args.<key>` only cares about the passed argument, so in this case we'd lose the config parsing and the environment. (`automagic_keyboard`).

```suggestion
    while not cli.config.list_keymaps.keyboard:
```